File: app/src/main/java/ru/magflayer/spectrum/presentation/pages/splash/SplashView.java
package ru.magflayer.spectrum.presentation.pages.splash;

import com.arellomobile.mvp.viewstate.strategy.OneExecutionStateStrategy;
import com.arellomobile.mvp.viewstate.strategy.StateStrategyType;

import ru.magflayer.spectrum.presentation.common.mvp.view.BaseView;

interface SplashView extends BaseView {

    @StateStrategyType(OneExecutionStateStrategy.class)
    void closeScreen();

}

File: app/src/main/java/ru/magflayer/spectrum/presentation/pages/splash/SplashView.kt
package ru.magflayer.spectrum.presentation.pages.splash

import moxy.viewstate.strategy.OneExecutionStateStrategy
import moxy.viewstate.strategy.StateStrategyType
import ru.magflayer.spectrum.presentation.common.mvp.view.BaseView

interface SplashView : BaseView {

    @StateStrategyType(OneExecutionStateStrategy::class)
    fun openMainScreen()

}
